src/lang/pt.json

```diff
@@ -200,31 +200,31 @@
   "sta.apps.rolldice": "Rolar Dados",
   "sta.apps.dedicatedfocus": "Usando Foco Dedicado?",
   "sta.apps.staroller": "STARoller",
-  "sta.apps.addmomentum": "adicionou {0} momentum ao pool",
-  "sta.apps.removemomentum": "removeu {0} momentum do pool",
-  "sta.apps.addthreat": "adicionou {0} ameaça ao pool",
-  "sta.apps.removethreat": "removeu {0} ameaça do pool",
+  "sta.apps.addmomentum": "adicionou {0} de Ímpeto à reserva",
+  "sta.apps.removemomentum": "removeu {0} de Ímpeto da reserva",
+  "sta.apps.addthreat": "adicionou {0} de Ameaça à reserva",
+  "sta.apps.removethreat": "removeu {0} de Ameaça da reserva",
 
   "sta.roll.success": "Sucesso",
   "sta.roll.successPlural": "Sucessos",
   "sta.roll.task.name": "Tarefa",
   "sta.roll.challenge.name": "Desafio",
-  "sta.roll.challenge.weaponreroll": "{name} Reroll",
-  "sta.roll.target": "Alvo",
+  "sta.roll.challenge.weaponreroll": "{name} Re-rolar",
+  "sta.roll.target": "Alvo:",
   "sta.roll.complicationrange": "Margem de Complicação é ",
-  "sta.roll.complication": "Houve uma Complicação!",
-  "sta.roll.complicationPlural": "Tivemos |#| Complicações!",
-  "sta.roll.effect": "Houve um efeito!",
-  "sta.roll.effectPlural": "Tivemos |#| efeitos!",
+  "sta.roll.complication": "Ocorreu uma Complicação!",
+  "sta.roll.complicationPlural": "Ocorreram |#| Complicações!",
+  "sta.roll.effect": "Ocorreu um efeito!",
+  "sta.roll.effectPlural": "Ocorreram |#| efeitos!",
   "sta.roll.rerollresults": "Re-rolar resultados?",
   "sta.roll.npccrew": "PdM Tripulante",
   "sta.roll.npccrewbasic": "Básica",
   "sta.roll.npccrewproficient": "Proficiente",
   "sta.roll.npccrewtalented": "Talentosa",
   "sta.roll.npccrewexceptional": "Excepcional",
   "sta.roll.npcship": "PdM Nave",
-  "sta.roll.shipassisting": "A Nave está assistindo?",
-  "sta.roll.complicationroller": "Margem de Complicações",
```

Comment:
This one needs to be complicationroller as there is already a complicationrange value in use above.